.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI tests
+
+on: [push, pull_request]
+
+jobs:
+
+  Run-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8']
+    steps:
+
+    - uses: actions/checkout@v1
+      with:
+        submodules: recursive
+
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install
+      run: |
+        pip install -r requirements.txt
+
+    - name: Check formatting
+      run: source .github/workflows/format.sh
```

Comment:
Should these be run, not sourced?

Reply:
This is following the pattern from https://github.com/SymbiFlow/symbiflow-xc-fasm2bels/blob/master/.github/workflows/ci.yml

I don't believe it matters that much source vs invoke.

Reply:
`source` means things like the `set` settings persist after the script ends.